tests/queries/0_stateless/03203_system_query_metric_log.sh

```diff
@@ -33,36 +33,6 @@ function check_log()
         WHERE event_date >= yesterday() AND query_id = '${query_prefix}_${interval}'
     """
 
-    # We calculate the diff of each row with its previous row to check whether the intervals at
-    # which data is collected is right. The first row is always skipped because the diff with the
-    # preceding one (itself) is 0. The last row is also skipped, because it doesn't contain a full
-    # interval. We leave at least 60% of margin for many rows and at most 80% for one single row.
```

Comment:
Nice try, but Linux is not a real-time operating system.

Reply:
Yep, even though I knew this was not deterministic, I didn't expect it to fail very often, to be honest. See the comment at the top:

> This test depends A LOT on timing, so it's very sensitive when the system is overloaded. For that reason, margins of 20% were added initially. They've been increased over time in an attempt to make it more stable, even though it's never going to be deterministically perfect.